index.html

```diff
@@ -308,62 +195,145 @@ <h3>What are Internationalization and Localization?</h3>
+        <p>A <dfn title="language-priority-list">language priority list</dfn>
+          is a collection of one or more language ranges identifying the user's
+          language preferences for use in matching. As the name suggests, such
+          lists are normally ordered or weighted according to the user's
+          preferences. The HTTP [[RFC2616]] <code>Accept-Language</code>[[RFC3282]]
+          header is an example of one kind of language priority list.</p>
+        <p>A <dfn title="basic-language-range">basic language range</dfn> is
+          simply a language tag used to express a language preference. An <dfn
+
+            title="extended-language-range">extended language range</dfn> allows
+          a more expressive set of language preference through the use of a
+          wildcard subtag <q><code>*</code></q>.</p>
+        <aside class="example">
+          <p>Basic versus extended language range and language priority list</p>
+          <p>The string <code>de-de</code> is a basic language range. It
+            matches, for example, the language tag <code>de-DE-1996</code>, but
+            not the language tag <code>de-Deva</code>.</p>
+          <p>The string <code>de-*-DE</code> is an extended language range. It
+            matches all of the following tags:</p>
+          <ul>
+            <li>
+              <p><code>de-DE</code></p>
+            </li>
+            <li>
+              <p><code>de-DE-x-goethe</code></p>
+            </li>
+            <li>
+              <p><code>de-Latn-DE-1996</code></p>
+            </li>
+          </ul>
+          <p><code>"en; fr; zh-Hant"</code> is a language priority list. It
+            would be read as "<span class="quote">English before French before
+              Chinese as written in the Traditional script</span>". Note that
+            the syntax shown is only an example, since it depends on the
+            protocol, application, or implementation that uses the list.</p>
+        </aside>
+      </section>
+      
+
+      <section id="best-practices">
+        <h2>Best Practices and Recommendations</h2>
+        
+        <p>This section provides specification authors and implementers with best practices recommended by the Internationalization (I18N) Working Group. These (and many other) best practices, along with links to supporting materials, can also be found in the <cite>Internationalization Best Practices for Spec Developers</cite> [[INTERNATIONAL-SPECS]]. In addition to the best practices found here, additional best practices relating to language metadata on the Web can be found in [[STRING-META]].</p>
+        
+        <aside class="note">
+           <p>In this section [[RFC2119]] keywords have their usual meaning. We differentiate <em>best practices</em>, which should be adopted by all specifications and <em>recommendations</em>, which require additional standardization or which are speculative prior to adoption.</p>
```

Comment:
Should we update the reference to BCP 14? (See an example here: https://www.w3.org/TR/appmanifest/#conformance )

Reply:
Respec does not know  BCP14. I can add a bibxml reference to our document later. I filed an issue to track it.